neuro_san/session/http_service_agent_session.py

```diff
@@ -135,9 +36,9 @@ def function(self, request_dict: Dict[str, Any]) -> Dict[str, Any]:
                     protobufs structure. Has the following keys:
                 "function" - the dictionary description of the function
         """
-        path: str = self._get_request_path("function")
+        path: str = self.get_request_path("function")
```

Comment:
Now that we are a proper subclass worth noting that the 2 _private members are now public.